docs/howtos/layers/labels.md

```diff
@@ -113,6 +113,22 @@ layer button above the layers list. The shape of the new labels layer will match
 the size of any currently existing image layers, allowing you to paint on top of
 them.
 
+## Saving labels layers
+
+When saving a labels layer, file size can be unnecessarily large when compression
+is not used.  By default, when saving a layer through the GUI no file 
+compression is applied.  The following commands will save a labels layer using 
```

Comment:
Wait, is that true??? If so in my opinion the fix is to use `compress=1` in the GUI also rather than writing these docs! 😂 

But I made sure compression was on by default in #3040, and that code [is still there](https://github.com/napari/napari/blob/4aef733d133f597c8ba12c7e47e9039f10d8fb17/napari/utils/io.py#L63-L68). But I see that it only applies to tiff files. Perhaps you encountered the issue with png? Personally I would prefer adding another 'elif' clause to that function to special-case png as well and make sure we always compress by default.